package.json

```diff
@@ -3,18 +3,20 @@
   "version": "1.3.1",
   "types": "types/index.d.ts",
   "main": "dist/index.js",
+  "module": "dist/esm/index.js",
```

Comment:
Is the PR title correct? I see that this PR adds `esm` build, thus a more clear title could be "Add esm build".

Reply:
updated 👍